src/components/chat-item/chat-prompt-input.ts

```diff
@@ -310,6 +311,55 @@ export class ChatPromptInput {
     }
   };
 
+  private readonly handleInputFocus = (): void => {
+    const inputValue = this.promptTextInput.getTextInputValue();
+    if (inputValue.startsWith('/') || inputValue.startsWith('@')) {
+      this.quickPickType = inputValue.startsWith('@') ? 'context' : 'quick-action';
+      const quickPickItems =
+        this.quickPickType === 'context'
+          ? MynahUITabsStore.getInstance().getTabDataStore(this.props.tabId).getValue('contextCommands') as QuickActionCommandGroup[]
+          : MynahUITabsStore.getInstance().getTabDataStore(this.props.tabId).getValue('quickActionCommands') as QuickActionCommandGroup[];
+      this.quickPickItemGroups = [ ...quickPickItems ];
+      this.quickPickTriggerIndex = inputValue.startsWith('@') ? inputValue.indexOf('@') : 1;
+      this.textAfter = inputValue.substring(this.quickPickTriggerIndex);
+      this.promptTextInput.setContextReplacement(this.quickPickItemGroups.length > 0);
+      const restorePreviousFilteredQuickPickItemGroups: QuickActionCommandGroup[] = [];
+      this.quickPickItemGroups.forEach((quickPickGroup: QuickActionCommandGroup) => {
+        const newQuickPickCommandGroup = { ...quickPickGroup };
+        try {
+          const searchTerm = inputValue.substring(this.quickPickTriggerIndex).match(/\S*/gi)?.[0];
+          const promptRegex = new RegExp(searchTerm ?? '', 'gi');
+          newQuickPickCommandGroup.commands = newQuickPickCommandGroup.commands.filter(command =>
+            command.command.match(promptRegex)
+          );
+          if (newQuickPickCommandGroup.commands.length > 0) {
+            restorePreviousFilteredQuickPickItemGroups.push(newQuickPickCommandGroup);
+          }
+        } catch (e) {
+          // In case the prompt is an incomplete regex
```

Comment:
is there a logging mechanism to not swallow this silently?

Reply:
Not yet, will be there in the future if it is being enabled from the configs.